tests/ssl-opt.sh

```diff
@@ -14086,6 +14086,28 @@ run_test    "Handshake defragmentation on client: len=5, TLS 1.3" \
             -c "handshake fragment: 0 \\.\\. 5 of [0-9]\\+ msglen 5" \
             -c "waiting for more fragments (5"
 
+requires_openssl_3_x
+requires_protocol_version tls13
+requires_certificate_authentication
+run_test    "Handshake defragmentation on client: len=3, TLS 1.3" \
```

Comment:
Please document this in the ChangeLog entry: "The initial fragment for each handshake message must be at least 4 bytes."